### ISSUE TITLE
Method invocation failed because [Deserialized.Microsoft.Dynamics.Nav.Apps.Types.AppId] does not contain a method named 'ToLowerInvariant'

### ISSUE THREAD

**Issue by ismailhafid1** [microsoft/navcontainerhelper#3377]
**Describe the issue**
After Updating the system files ,i ran into an error during Next Major Workflow's execution  :
` Unexpected error when running action. Error Message: Method invocation failed because [Deserialized.Microsoft.Dynamics.Nav.Apps.Types.AppId] does not contain a method named 'ToLowerInvariant'., StackTrace: at <ScriptBlock>, C:\ProgramData\BcContainerHelper\6.0.6\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 1857 <- at <ScriptBlock>, C:\ProgramData\BcContainerHelper\6.0.6\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 1825 <- at <ScriptBlock>, C:\ProgramData\BcContainerHelper\6.0.6\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 1477 <- at <ScriptBlock>, C:\ProgramData\BcContainerHelper\6.0.6\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 1470 <- at <ScriptBlock>, C:\ProgramData\BcContainerHelper\6.0.6\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 948 <- at Run-AlPipeline, C:\ProgramData\BcContainerHelper\6.0.6\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 910 <- at <ScriptBlock>, 
`

**Full output of scripts**
```
BcContainerHelper version 6.0.6
BC.HelperFunctions emits usage statistics telemetry to Microsoft
```

**Additional context**
- does it happen all the time? : Yes , for NextMajor workflow. Starting sometime in the last week
- did it use to work? : Yes ,( two weeks ago ), the next Major Workflow worked successfully

```[tasklist]
### Tasks
```

**Comment by freddydk** [comment]
This is a change in NextMajor, which is fixed in the preview of BcContainerHelper.
Please try installing the preview

**Comment by freddydk** [comment]
BCContainerHelper 6.0.7 has shipped with a fix for this (with generic images 1.0.2.15 or above)
Also note that generic images 1.0.2.16 for the latest Windows versions has shipped.
